### PR TITLE
♻️ cavatica refactor

### PR DESCRIPTION
**Pull request by hlminh2000**
1) some refactoring of cavatica stuff into service
2) tree shake
3) some async performance improvement

### CODE REVIEW

File: src/components/UserDashboard/CavaticaProjects/CavaticaProvider.js [hunk 1]
```diff
@@ -10,22 +10,24 @@ const CavaticaProvider = ({ children, onData }) => (
...
-        const failedTasks = await getTasks({ project: p.id, type: 'FAILED' });
-        const runningTasks = await getTasks({ project: p.id, type: 'RUNNING' });
+      Promise.all(
+        projects.map(async p => {
+          const [members, completedTasks, failedTasks, runningTasks] = await Promise.all([
+            getMembers({ project: p.id }),
+            getTasks({ project: p.id, type: 'COMPLETED' }),
+            getTasks({ project: p.id, type: 'FAILED' }),
+            getTasks({ project: p.id, type: 'RUNNING' }),
+          ]);
```
Comment (hlminh2000) [inline src/components/UserDashboard/CavaticaProjects/CavaticaProvider.js]: @ciaranschutte for mutually independent async operations, this should help speed things up a little :)

File: src/uikit/Multicard/index.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-    y: 0,
-    opacity: 1,
-  },
-  exit: {
-    y: 50,
-    opacity: 0,
-    transition: { duration: 150 },
-  },
-});
+// const AnimatedChild = posed.div({
```
Comment (joneubank) [inline src/uikit/Multicard/index.js]: Remove instead of comment out?

File: src/uikit/Multicard/index.js [hunk 3]
```diff
@@ -36,16 +36,18 @@ class Multicard extends Component {
     this.setIndex = this.setIndex.bind(this);
   }
 
+  /*
```
Comment (joneubank) [inline src/uikit/Multicard/index.js]: Remove instead of comment?

File: src/uikit/Multicard/index.js (hunk 2 above)
Comment (hlminh2000) [reply src/uikit/Multicard/index.js]: I think Ciaran wanted these around so he can revisit to add some animation :P I'm just silencing the linters

**Pull request merged**